prow/test-infra-cleanup-GKE.sh

```diff
@@ -27,4 +27,4 @@ set -u
 set -x
 
 echo "=== Cleaning up GKE clusters ==="
-./scripts/cleanup-GKE "$@"
+./scripts/cleanup-GKE -h 1
```

Comment:
I thought we agree 2 hours but I guess 1 is fine :)

Reply:
the problem is that the e2e-smoke also creates cluster during presubmit, and they run more ofter so I rather have 1 h